components/Shared/MessageHistoryTable/MessageHistoryRowContainer.js

```diff
@@ -9,11 +9,14 @@ export default styled(Box).attrs(props => ({
   my: 1,
   ...props
 }))`
+  background-color: ${props => props.theme.colors.input.background.base}00;
```

Comment:
@as-dr my one question - what is the `00` at the end here?

Reply:
It sets the opacity to zero. Opacity is one of only a handful of performant animations we can use (i.e. that don't require a repainting of the DOM). This is a better way to implement a background color change vs. adding a background on `hover`